build/protobuf.mk

```diff
@@ -39,8 +39,8 @@ GOGOPROTO_PATH  := $(GOGOPROTO_ROOT):$(GOGOPROTO_ROOT)/protobuf
 
 COREOS_PATH := $(GITHUB_ROOT)/coreos
 
-GO_PROTOS  := $(sort $(shell find $(REPO_ROOT) -not -path '*/.*' -name *.proto -type f))
-GO_SOURCES  := $(GO_PROTOS:%.proto=%.pb.go)
+GO_PROTOS := $(addprefix $(REPO_ROOT)/, $(sort $(shell cd $(REPO_ROOT) && git ls-files --exclude-standard --cached --others -- '*.proto')))
```

Comment:
Minor tip: you can pass `git -C $(REPO_ROOT)` instead of running `cd` in a subshell.